doc/developer/vtysh.rst

```diff
@@ -178,7 +178,7 @@ message from VTYSH to a daemon:
    00000010: 6c0a 00                                  l..
 
 
-The response format has some more data in it. First is a NULL-terminated string
+The response format has some more data in it. First is a NUL-terminated string
```

Comment:
"NULL" is pretty familiar - any specific reason to change it?

Reply:
I guess both are familiar to most developers, just trying to be correct. I didn't give it more than 10 seconds though to be honest.

Reply:
I guess my concern is that I'd like to avoid a situation where we run some tool now and make a change, then in a year someone runs another tool and we go through the process and change something back.

Reply:
Agreed, but in any case we should pick one or the other.
A grep through the tree shows all code uses NUL-terminated, apart from these two in this document.

Reply:
No, I think Mark is right on this one. We need to keep it as NULL. I'm not sure what you're referring to by the rest of the tree.

```
vagrant@ubuntu-bionic [master] ~/frr> rg "NUL[^L]" | wc -l
19
vagrant@ubuntu-bionic [master] ~/frr> rg "NULL" | wc -l
11714
```

Reply:
I meant this:
```
[ruben@rubenkerkhof frr]$ git grep NUL-terminated
lib/buffer.h:/* Add a NUL-terminated string to the end of the buffer. */
lib/buffer.h:   single NUL-terminated string allocated using XMALLOC(MTYPE_TMP).  Note
lib/command.h:/* Concatenates argv[shift] through argv[argc-1] into a single NUL-terminated
zebra/interface.h:      /* Domain name without trailing root zone dot (NUL-terminated) */
```
vs
```
[ruben@rubenkerkhof frr]$ git grep NULL-terminated
doc/developer/vtysh.rst:NULL-terminated strings, whose content are CLI commands. Here is a typical
doc/developer/vtysh.rst:The response format has some more data in it. First is a NULL-terminated string
```

Reply:
Ah, I see where you're coming from. Well, the C constant is named NULL; I'd rather change `buffer.h` in this case.

Reply:
I really don't have a strong preference for one or the other, I'll drop this specific change.